### PR TITLE
Added extension check that returns an error if submit data or send me…

### PR DESCRIPTION
**Pull request by markopoloparadox**
Companion PR: https://github.com/availproject/avail-core/pull/83
resolves https://github.com/availproject/engineering/issues/540

### CODE REVIEW

**Comment by Leouarz** [comment]
Isn't this possible without adding a new extension ?

File: pallets/transaction-payment/src/lib.rs [hunk 1]
```diff
@@ -1048,3 +1048,13 @@ where
 		Self::compute_actual_fee(len, &info, &post_info, Zero::zero())
 	}
 }
+
+pub struct TestLengthToFeeU64;
```
Comment (Leouarz) [inline pallets/transaction-payment/src/lib.rs:1052]: Why is this needed ?

**Review by ToufeeqP — APPROVED** [review]

File: pallets/transaction-payment/src/lib.rs (hunk 1 above)
Comment (markopoloparadox) [reply pallets/transaction-payment/src/lib.rs:1052]: It's used in `pallets/dactr/src/extensions/check_batch_transactions_mock.rs`.

Basically we cannot impl `WeightToFee` trait for `u64` so we need to create a wrapper first. 
Instead of every mock creating the same wrapper, I just created and placed a default test one in the pallet `lib.rs` so that it can be used everywhere.

**Review by Leouarz — APPROVED** [review]
